File: 3110: scoreOfAString.cs
public class Solution {
    public int ScoreOfString(string s) {
        
        List<int> vals = new List<int>();
        int res = 0;
        for(int i = 0; i < s.Length-1; i++){

            int c1 = s[i];
            int c2 = s[i+1];
            vals.Add(Math.Abs(c1-c2));
        }
        
        for(int i = 0; i < vals.Count; i++){

            res += vals[i];
        }
        return res;
    }
}

File: 3110: scoreOfAString.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        vals = []
        res = 0
        for i in range(len(s)-1):
            vals.append(abs(ord(s[i]) - ord(s[i+1])))
        for i in vals:
            res += i
        return res